Readme.md

```diff
@@ -178,6 +184,11 @@ Wait until the element `selector` is present e.g. `.wait('#pay-button')`
 #### .wait(fn[, arg1, arg2,...])
 Wait until the `fn` evaluated on the page with `arg1, arg2,...` returns `true`. All the `args` are optional. See `.evaluate()` for usage.
 
+#### .wait('downloads-complete')
+Wait until all downloads are in a state of `'completed'`, `'interrupted'`, or `'cancelled'`
```

Comment:
Fair enough.  What do you propose as an alternative?  Should there be a standalone method, eg `.waitForDownloads()`, or should the wait string start with a character that is illegal for tags, eg `.wait(':downloads-complete')`?